index.rst

```diff
@@ -2,24 +2,206 @@
+
+A CI/CD system is an integral part of Telescope and Site Software deliverables, alongside a suite of integration tests, and a reporting system that are designed to verify the system integrity after every new release and for daily checkouts prior to night-time observations.
+
+We plan to use this suite of integration tests to verify the majority of Telescope and Site Software requirements, alongside any affiliated requirements that can be verified in the process and use the reporting system to generate verification artifacts.
+In the following sections we will go over the details of the different steps in the testing process and reporting products.
+
+Testing Suite
+=============
+
+In this section, we discuss the collection of tests that are most relevant for the formal requirement verification.
+.. Still, as we mentioned earlier, there are a whole selection of unit tests that are used early on in the development process to safeguard the integrity of each software components.
+It should be remembered that there are minimal functionality tests and other categories of tests completed at various stages of the system lifecycle to safeguard the integrity of each component in addition to the tests discussed in this section.
+
+SAL Scripts
+-----------
+
+SAL Scripts are specialized components designed to execute short lived actions in the observatory control system, including, but not limited to; enabling components, preparing the system for calibrations, taking calibration, tracking targets and taking data, etc.
+In principle, most operations executed by the system will be performed using the ScriptQueue, by SAL Scripts.
+
+By default, these components are developed using the BaseScript Python class provided by `salobj`_.
+They are developed in one of the two available repositories; `ts_externalscripts`_ and `ts_standardscripts`_, and each change to the code undergoes a peer review process to ensure TSSW development guidelines have been followed.
+This includes ensuring that sufficient unit tests are present.
+See :tstn:`010` for details on the use case for each of these repositories.
+
+.. _salobj: ts-salobj.lsst.io
+.. _ts_externalscripts: https://github.com/lsst-ts/ts_externalscripts
+.. _ts_standardscripts: https://github.com/lsst-ts/ts_standardscripts
+
+The usage of SAL Scripts for integration testing comes, mainly, in two different varieties:
+
+#.  Scripts used in standard operations that can be executed in an environment where the definition of success can be explicitly defined (e.g. track_target_and_take_image).
+    These can also be used to test fault scenarios by running the script in a mode that we know will cause a particular condition.
+    For example, use the ``track_target_and_take_image`` script in a situation that would cause the system to run into a limit while taking data.
+#. A custom made script to test a particular condition (e.g. forcing a fault condition).
```

Comment:
I think you need to add an example here since you also talk about forcing a fault condition in the above sentence... or just change the example to something not involving faults that might be self explanatory?
Maybe a series of non-normal hexapod motions to test against overheating? You can probably come up with something better...